### ISSUE TITLE
Bad dependency specification allows for update to Beet version with breaking change.

### ISSUE THREAD

**Issue by Luexa** [Luewd/lepsen-core#1]
The intent of the broad dependency specification for Beet was to allow dependent packages to use newer versions of Beet that didn't have any breaking changes, and they could just have a stricter dependency specification or rely on `poetry.lock`, but from personal experience this turned out to be a really bad idea.

This project does not support the latest version of Beet, but I am not fully aware of exactly what is broken nor am I prepared to update the dependent package in which I experienced this issue, hence I opened this issue to track this oversight (which sadly made it into PyPI `lepsen_core`).

The issue I immediately noticed was that `hangman("*")` is no longer valid syntax, but I am unsure of the correct syntax, in case anyone else wants to comment on this issue. Of course, that was just the first error I noticed; I didn't try to debug it, so who knows how many other things about the plugin are broken in Beet 0.35. For now the workaround is to just require Beet `^0.33.0` in dependent packages.

**Comment by vberlier** [comment]
Just letting you know that apart from built-in plugins switching over to `@configurable` there shouldn't be any other breaking change. Plugins like hangman can now be used like this:

```python
from beet import Context
from beet.contrib.hangman import hangman

def beet_default(ctx: Context):
    ctx.require(hangman(match=["*"]))
```

**Comment by Luexa** [comment]
The suggested fix has been applied to v0.1.1. There are no changes to the generated data pack, only to the Beet plugin and project dependency specification.
